src/mscorlib/shared/System/NullableExtensions.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System
+{
+    public static class NullableExtensions
```

Comment:
If this was going to be an extension method, there's already the static `System.Nullable` class that could be used.